# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../pdfedit/hotfix.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- pdfedit-0.4.5/src/xpdf/xpdf/Object.h.vanilla	2019-08-09 19:24:02.246642346 +0200
+++ pdfedit-0.4.5/src/xpdf/xpdf/Object.h	2019-08-09 19:24:07.738642587 +0200
@@ -206,7 +206,7 @@
 private:
 
   ObjType type;			// object type
-  mutable union {		// value for each type:
+  union {		// value for each type:
     GBool booln;		//   boolean
     int intg;			//   integer
     double real;		//   real
